sktime/transformations/series/tests/test_window_summarizer.py

```diff
@@ -76,6 +76,14 @@ def count_gt100(x):
         count_gt100: [[3, 2]],
     }
 }
+
+kwargs_series = {
```

Comment:
I would propose to rename these kwargs to `kwargs_negative_lags` since they could also be applied to hierarchical data etc.